docs/source/install.md

```diff
@@ -0,0 +1,85 @@
+# Install Fabric and Fabric Samples
+
+Please install the [Prerequisites](./prereqs.html) before following these install instructions.
+
+We think the best way to understand something is to use it yourself.  To help you use Fabric, we have created a simple Fabric network called `test-network` using docker compose, and a set of sample applications that demonstrate its core capabilities.
+We have also precompiled `Fabric CLI tool binaries` and `Fabric Docker Images` which will be downloaded to your environment, to get you going.
+
+The cURL command in the instructions below sets up your environment so that you can run the 'test-network'. Specifically, it performs the following steps:
+* Clones the [hyperledger/fabric-samples](https://github.com/hyperledger/fabric-samples) repository.
+* Downloads the latest Hyperledger Fabric Docker images and tags them as `latest`
+* Downloads the following platform-specific Hyperledger Fabric CLI tool binaries and config files into the `fabric-samples` `/bin` and `/config` directories. These binaries will help you interact with the 'test-network'.
```

Comment:
Should test network be in backticks to match style at the top. I do not have any preference (other docs just call it the Fabric test network), but we should be consistant.

Reply:
I've made it Fabric test network consistently.